packages/react-components/react-charts-preview/stories/LineChart/LineChartEvents.stories.tsx

```diff
@@ -4,7 +4,8 @@ import { DataVizPalette } from '../../src/utilities/colors';
 import { mergeStyles } from '@fluentui/react/lib/Styling';
 import * as d3 from 'd3-format';
 import { Switch } from '@fluentui/react-components';
-import { Checkbox } from '@fluentui/react/lib/Checkbox';
+import { Checkbox } from "@fluentui/react-components";
+import type { CheckboxOnChangeData, CheckboxProps } from "@fluentui/react-components";
 import { ColorPicker, IColor } from '@fluentui/react';
```

Comment:
remove color picker. It is also a v8 control

Reply:
This has been replaced with SwatchPicker

Reply:
Okay okay, did not remove import of colorpicker. will remove that